.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -7,8 +7,8 @@ contact_links:
     url: https://github.com/uBlockOrigin/uAssets/issues
     about: Report issues with filter lists or broken website functionality in the uAssets issue tracker.
   - name: uBO Lite (uBOL) Issues
-    url: https://github.com/uBlockOrigin/uBOL-issues/issues
+    url: https://github.com/uBlockOrigin/uBOL-home/issues
     about: Report issues specific to the Manifest Version 3 (MV3) variant in the uBOL issue tracker.
-  - name: Filter List Requests
-    url: https://github.com/uBlockOrigin/uBlock-issues#filter-list-requests
+  - name: Filter List Issues
```

Comment:
Based on the description, it seems this is about requests to add new filterlists to uBlock Origin, especially given line #6 concerns issues with filterlists.
![An issue option titled "Filter List Issues", with the description "New filter list requests are not permitted and will be declined and closed."](https://github.com/user-attachments/assets/4aa5ad29-cac4-4938-8252-b06b5b63ad11)
Thanks